Sources/IO/Misc/PDBReader/index.js

```diff
@@ -238,7 +190,6 @@ export function extend(publicAPI, model, initialValues = {}) {
     'xSpacing',
```

Comment:
Yes at the beginning it was copied from elevator reader and I thought it was maybe an important parameter but maybe not finally. We should ask @cryos.